pipeline/daily/stages.groovy

```diff
@@ -64,7 +66,17 @@ python host_os.py    \
            --updater-email $params.GITHUB_BOT_EMAIL \
            --push-repo-url $VERSIONS_PUSH_REPO_URL \
            --push-repo-branch $COMMIT_BRANCH \
-"""
+""", returnStatus: true)
+
+    Integer SUCCESS_EXIT_CODE = 0
+    Integer NO_UPDATES_EXIT_CODE = 56
+    if (exitCode == SUCCESS_EXIT_CODE) {
+      updateSuccessful = true
+    } else if (exitCode == NO_UPDATES_EXIT_CODE) {
+      updateSuccessful = false
+    } else {
+      error('Packages update failed')
+    }
```

Comment:
We don't want to raise an error when NO_UPDATES_EXIT_CODE is returned. We don't want to do anything in that case. I just left the attribution there to make it clearer that the update did not happen, but I could leave it empty.